src/cljam/algo/pileup.clj

```diff
@@ -110,12 +111,29 @@
 (defn filter-by-base-quality
   "Returns a predicate for filtering piled-up reads by base quality at its
   position."
-  [min-base-quality]
+  [^long min-base-quality]
   (fn [p]
     (->> #(<= min-base-quality (.qual ^PileupBase %))
          (partial filterv)
          (update p 1))))
 
+(defn- unzip-2
+  [transform-fn]
+  (fn [rf]
+    (let [va (volatile! (transient []))
+          vb (volatile! (transient []))]
+      (fn
+        ([] (rf))
+        ([acc]
+         (-> acc
+             (rf (persistent! @va))
+             (rf (persistent! @vb))))
```

Comment:
arity-1 of a transducer should call arity-1 of `rf`

cf. transducing with a reducing function like the following.
```clojure
(fn ([] (transient []))([acc] (persistent! acc))([acc x] (conj! acc x)))
```

Reply:
Fixed